omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -68,8 +68,11 @@
 default['private_chef']['addons']['ubuntu_supported_codenames'] =
```

Comment:
@PrajaktaPurohit I'm real confused by this part of the cookbook code:

Lucid: 10.04
Precise: 12.04
Trusty: 14.04

All of those are EOL and will never be encountered so this logic means we're always going to fill in Ubuntu 10.04, which seems very wrong. I'm also not seeing where this attribute is actually used in the cookbook anymore.

Reply:
Good catch! That should point to the latest.